File: mvpcore/src/main/java/biz/dealnote/mvp/reflect/OnGuiCreated.java
package biz.dealnote.mvp.reflect;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by ruslan.kolbasa on 05.10.2016.
 * phoenix
 */
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.METHOD)
public @interface OnGuiCreated {
}

File: mvpcore/src/main/java/biz/dealnote/mvp/reflect/OnGuiCreated.kt
package biz.dealnote.mvp.reflect

/**
 * Created by ruslan.kolbasa on 05.10.2016.
 * phoenix
 */
@Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER)
annotation class OnGuiCreated